docs/docs/concepts/foundation/transactions.md

```diff
@@ -11,13 +11,49 @@ On this page you'll learn:
 - The Aztec Kernel and its two circuits: private and public, and how they execute function calls
 - The call stacks for private & public functions and how they determine a transaction's completion
 
+## Simple Example of the (Private) Transaction Lifecycle
+
+The transaction lifecycle for an Aztec transaction is fundamentally different from the lifecycle of an Ethereum transaction.
+
+The introduction of the Private eXecution Environment (PXE) provides a safe environment for the execution of sensitive operations, ensuring that decrypted data are not accessible to unauthorized applications. However, the PXE exists client-side on user devices, which creates a different model for imagining a transaction lifecycle. The existence of a sequencing network also introduces some key differences between the Aztec transaction model and the transaction model used for other networks.
+
+The accompanying diagram illustrates the flow of interactions between a user, their wallet, the PXE, the node operators (sequencers / provers), and the L1 chain.
+
+<Image img={require("/img/transaction-lifecycle.png")} />
```

Comment:
nice image but can you also clearly indicate what happens on user's device  vs what doesn't?